FerramAerospaceResearch/Settings/OcclusionSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FerramAerospaceResearch.Reflection;
+
+namespace FerramAerospaceResearch.Settings
+{
+    [ConfigNode("Occlusion", shouldSave: false, parent: typeof(FARConfig))]
+    public static class OcclusionSettings
+    {
+        [ConfigValue("useRaycaster")] public static bool UseRaycaster { get; set; } = false;
+        [ConfigValue("jobsPerThread")] public static int JobsPerThread { get; set; } = 3;
+        [ConfigValue("maxJobs")] public static int MaxJobs { get; set; } = 10;
+        [ConfigValue("fibonacciLatticeSize")] public static int FibonacciLatticeSize { get; set; } = 2000;
+        // 1000 points produces typical misses 2-3 degrees when optimized for average miss distance.
+        [ConfigValue("maxRaycastDimension")] public static int MaxRaycastDimension { get; set; } = 100;
+        [ConfigValue("raycastResolution")] public static float RaycastResolution { get; set; } = 0.1f;
+        [ConfigValue("resetInterval")] public static float ResetInterval { get; set; } = 180;
```

Comment:
Should probably hook into voxelization and reset when voxelization is updated only

Reply:
I thought about that.  There's a couple conflicting ideas.
One is... adjacent vessels can block the raycasts.  For instance, after a staging event.  So there's some risk of stale data that I'd like to regenerate.  (I also didn't want to explode the length of the RaycastHit returned array to search for something on the current vessel.  It's also slightly neat that I -can- allow some other vessel to block, though as mentioned that has problems with long-lived data.)  I don't think FAR very often wants to voxelize unless there's a particular cause, ie staging or it is tracking part animations/deployment.
The other was just to exercise the thing while it's in dev.  That interval can always be made even sillier-long.

As a separate thing, I -am- reconsidering always running a Raycast for just the convection handling, and not using the cached area data.  More feasible now, having learned to use Burst.  The caching was a big performance aspect because the compute was so costly with just Jobs.  But that's very early WIP, where I'd like to do a little more interesting aerothermodynamics things.